r-package/grf/R/analysis_tools.R

```diff
@@ -50,6 +50,16 @@ get_tree <- function(forest, index) {
       paste("X", i, sep = ".")
     }
   })
+  
+  # for each node, calculate the leaf stats
+  tree$nodes <- lapply(tree$nodes, 
```

Comment:
This formatting seems a bit off, I think it would be more readable as follows:

```
tree$nodes <- lapply(tree$nodes, function(node) {
  if (node$is_leaf) {
    node$leaf_stats <- leaf_stats(forest, node$samples)
  }
  node
})
```